src/forms/modelForm.tsx

```diff
@@ -189,163 +178,194 @@ class ModelForm extends React.Component<ModelFormProps, ModelFormState> {
-          className="create-model-form bp3-form-group"
+          className="create-form bp3-form-group"
         >
-          <FormGroup className="suggest" label="Vendor (required)">
-            <StringSuggest
-              inputProps={{
-                placeholder: "vendor",
-              }}
-              popoverProps={{
-                minimal: true,
-                popoverClassName: "dropdown",
-                usePortal: true,
-              }}
-              defaultSelectedItem={this.state.values.vendor}
-              inputValueRenderer={(vendor: string) => this.state.values.vendor}
-              items={this.state.vendors}
-              onItemSelect={(vendor: string) => {
-                console.log("item selected ");
-                this.setState({
-                  values: updateObject(values, { vendor: vendor }),
-                });
-              }}
-              onQueryChange={(vendor: string) => {
-                console.log("CHANGE", vendor);
+          <Card>
+            <FormGroup className="suggest" label="Vendor*">
+              <StringSuggest
+                inputProps={{
+                  placeholder: "vendor",
+                }}
+                popoverProps={{
+                  minimal: true,
+                  popoverClassName: "dropdown",
+                  usePortal: true,
+                }}
+                defaultSelectedItem={this.state.values.vendor}
+                inputValueRenderer={(vendor: string) =>
+                  this.state.values.vendor
+                }
+                items={this.state.vendors}
+                onItemSelect={(vendor: string) => {
+                  this.setState({
+                    values: updateObject(values, { vendor: vendor }),
+                  });
+                }}
+                onQueryChange={(vendor: string) => {
+                  this.setState({
+                    values: updateObject(values, { vendor: vendor }),
+                  });
+                }}
+                itemRenderer={renderStringItem}
+                itemPredicate={filterString}
+                noResults={<MenuItem disabled={true} text="No results." />}
+              />
+            </FormGroup>
+
+            <FormGroup label="Model Number*" inline={false}>
+              <Field
+                placeholder="model_number"
+                onChange={this.handleChange}
+                value={values.model_number}
+                field="model_number"
+              />
+            </FormGroup>
+            <FormGroup label="Display Color">
+              <Field
+                field="display_color"
+                type="color"
+                value={values.display_color ? values.display_color : "#394B59"}
+                onChange={this.handleChange}
+              />
+            </FormGroup>
+          </Card>
+          <Card>
+            <RadioGroup
+              label="Mount Type*"
+              className="radio-group"
+              onChange={(e: any) => {
```

Comment:
This is picky but I'm not sure Mount Type is accurate bc both classic assets and blade chassis are rackmount.  Maybe model type?  Or is there another word that describes it better

Reply:
Discussed in standup that'll we'll leave it but ask Bletsch to confirm - are you gonna post on Piazza?

Reply:
yep